### PR TITLE
APPDEV-1225 Fabric reported crash in TimelineFragment 

### PR DESCRIPTION
**Pull request by Sivateja-CMI**
-- Setting the tag to list item holder view to differentiate the header and other views in.
-- Minor code refactoring.

APPDEV-1223 - Crash in header-- Adding reference to duplicate ticket.

### CODE REVIEW

File: app/src/main/java/nu/yona/app/ui/frinends/TimelineStickyAdapter.java [hunk 1]
```diff
@@ -77,13 +77,25 @@ public TimelineHolder onCreateViewHolder(ViewGroup parent, int viewType)
...
-		holder.getView().setTag(dayActivity);
+		setHolderViewTag(holder, dayActivity);
 		setUpTimeLineViewHolderWithDayActivity(holder, dayActivity);
 	}
 
+	public void setHolderViewTag(TimelineHolder holder, DayActivity dayActivity)
+	{
+		if (dayActivity.getChartTypeEnum() == ChartTypeEnum.TITLE)
+		{
+			holder.getView().setTag(headerViewTag);//to differentiate in item.onClickListener.
```
[the remark this answers is not in the document]
Comment (Sivateja-CMI) [reply app/src/main/java/nu/yona/app/ui/frinends/TimelineStickyAdapter.java]: I agree, at the moment we don not need to detect the click in header. I will update it.

File: app/src/main/java/nu/yona/app/ui/frinends/TimelineStickyAdapter.java [hunk 2]
```diff
@@ -33,7 +33,8 @@
 
 	private List<DayActivity> dayActivityList;
 	private final View.OnClickListener listener;
-	private Context mContext;
+	private Context context;
+	private static final String headerViewTag = "HeaderView";
```
[the remark this answers is not in the document]
Comment (Sivateja-CMI) [reply app/src/main/java/nu/yona/app/ui/frinends/TimelineStickyAdapter.java]: Agree to comment below. Removed constant.

File: app/src/main/java/nu/yona/app/ui/frinends/TimelineStickyAdapter.java (hunk 1 above)
[the remark this answers is not in the document]
Comment (Sivateja-CMI) [reply app/src/main/java/nu/yona/app/ui/frinends/TimelineStickyAdapter.java]: Done. Pushed update.

**Comment by Sivateja-CMI** [comment]
Yes bert, we should merge it. Accidental touch on headers can cause this crash.

**Comment by Bert-R** [comment]
OK, done. Build is running now.

**Pull request merged**